src/pages/search/advanced_search/PublisherFilters.tsx

```diff
@@ -0,0 +1,77 @@
+import { Autocomplete, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { t } from 'i18next';
+import { useState } from 'react';
+import { useHistory } from 'react-router';
+import { searchForPublishers } from '../../../api/publicationChannelApi';
+import { ResultParam } from '../../../api/searchApi';
+import { AutocompleteTextField } from '../../../components/AutocompleteTextField';
+import { Publisher } from '../../../types/registration.types';
+import { useDebounce } from '../../../utils/hooks/useDebounce';
+
+export const PublisherFilters = () => {
```

Comment:
Skal det inn flere filter her etter hvert?
Om ikke foreslår jeg:
```suggestion
export const PublisherFilter = () => {
```
Husk i så fall renaming av fil også